tests/ci_install.sh

```diff
@@ -1,37 +1,39 @@
 set -x   # Show which command is being run
 
-# geos is needed for cartopy, see
-# https://scitools.org.uk/cartopy/docs/latest/installing.html?highlight=install#building-from-source
-case ${RUNNER_OS} in
-linux|Linux)
-    sudo apt-get -qqy update
-    sudo apt-get -qqy install \
-      libhdf5-serial-dev \
-      libnetcdf-dev \
-      libgeos-dev \
-      libopenmpi-dev \
-      libfuse2
-    ;;
-osx|macOS)
-    sudo mkdir -p /usr/local/man
-    sudo chown -R "${USER}:admin" /usr/local/man
-    HOMEBREW_NO_AUTO_UPDATE=1 brew install hdf5 geos open-mpi netcdf ccache macfuse
-    ;;
-esac
-
 # Sets default backend to Agg
 cp tests/matplotlibrc .
 
 # Step 1: pre-install required packages
-if [[ "${RUNNER_OS}" == "Windows" ]] && [[ ${dependencies} != "minimal" ]]; then
-    # windows_conda_requirements.txt is a survivance of test_requirements.txt
-    # keep in sync: pyproject.toml
-    conda install --yes --file=tests/windows_conda_requirements.txt
-else
+if [[ ${dependencies} == "full" || ${dependencies} == "cartopy" ]]; then
     # upgrading pip to guarantee installing extra dependencies with [full] is supported
     # this is only necessary for some versions of Python 3.8 and 3.9
     # see https://github.com/yt-project/yt/issues/4270
-    python -m pip install --upgrade pip
+    python -m pip install 'pip>=21.2'
+
+    # see https://scitools.org.uk/cartopy/docs/latest/installing.html?highlight=install#building-from-source
+    if [[ "${RUNNER_OS}" == "Windows" ]]; then
+        conda install shapely scipy
+    else
```

Comment:
I'm not sure this windows case is needed at all anymore? we certainly shouldn't need to install shapely anymore since that installation step was needed to link to geos when building from source.